app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -34,12 +34,19 @@ def process_branch
   end
 
   def rubocop_comments
-    MessageBuilder.new(results, branch).comments
+    message_builder = MessageBuilder.new(results, branch)
+
+    {
+      :status   => message_builder.commit_status,
+      :comments => message_builder.comments
+    }
   end
 
   def replace_rubocop_comments
     logger.info("Updating PR #{pr_number} with rubocop comment.")
-    GithubService.replace_comments(fq_repo_name, pr_number, rubocop_comments) do |old_comment|
+    msg_builder_result = rubocop_comments
+
+    GithubService.replace_comments(fq_repo_name, pr_number, msg_builder_result[:comments], msg_builder_result[:status]) do |old_comment|
```

Comment:
Do you really need the 2 arguments if you're taking them from the same hash? Send the whole hash instead!